microsoft-edge/webview2/index.md

```diff
@@ -156,6 +156,7 @@ A General Availability \(GA\) or Preview version of WebView2 is available for th
 
 WebView2 apps can run on the following versions of Windows.
 
+*   Windows 11
 *   Windows 10
 *   Windows 10 IoT Enterprise LTSC x32 2019
 *   Windows 10 IoT Enterprise LTSC x64 2019
```

Comment:
In toc.yml, change 
Get started with remote debugging Windows 10 devices
to
Get started with remote debugging Windows devices
Android equivalent article doesn't specify version # in title.

Reply:
In toc.yml, 4 entries, delete "Get started with ".
Keep "Remote debugging with " on each item though; avoid the attempted strategy of having the user mentally piece together the parent node label with each child node's label.   
eg in the TOC, change:
Get started with remote debugging Windows devices
to:
Remote debugging Windows devices
or:
Remote debugging of Windows devices